### PR TITLE
feat(themes): add editor colors to platform theme

### PR DESCRIPTION
**Pull request by casey-speer**
### Overview
This PR adds editor colors to the `platformTheme` for use with the sandbox and future LE visual updates and updates the jest snapshot to match the changes.

Adds editor colors to the `platformTheme`.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/o7ZCG7KQ629ixpPuC9QYGo/Sandbox?node-id=739%3A194
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-styles/src/themes/platform.ts [hunk 1]
```diff
@@ -12,7 +12,35 @@ export const platformTheme = createTheme(coreTheme)
   .addColors(platformPalette)
   .addColorModes('dark', {
     light: {},
```
Comment (codecaaron) [inline packages/gamut-styles/src/themes/platform.ts:14]: Do we not have the light mode versions of these yet?

Comment (casey-speer) [reply packages/gamut-styles/src/themes/platform.ts:14]: We have colors but they're not finalized yet. I can add them but they will possibly change. Will consult with Sarah to see if there's any indication what the final colors will be.

**Review by slin12 — APPROVED** [review]

**Review by codecaaron — APPROVED** [review]
👍

File: packages/gamut-styles/src/themes/platform.ts (hunk 1 above)
Comment (casey-speer) [reply packages/gamut-styles/src/themes/platform.ts]: Light colors added!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.4-alpha.5b37cb.0<br/>@codecademy/gamut-illustrations@0.16.2-alpha.5b37cb.0<br/>@codecademy/gamut-kit@0.3.120-alpha.5b37cb.0<br/>@codecademy/gamut-labs@18.2.3-alpha.5b37cb.0<br/>@codecademy/gamut-patterns@0.2.3-alpha.5b37cb.0<br/>@codecademy/gamut-styles@10.1.3-alpha.5b37cb.0<br/>@codecademy/gamut-tests@2.4.3-alpha.5b37cb.0<br/>@codecademy/gamut@33.1.6-alpha.5b37cb.0<br/>@codecademy/konamimojisplosion@0.2.34-alpha.5b37cb.0<br/>@codecademy/markdown-overrides@0.5.46-alpha.5b37cb.0<br/>@codecademy/styleguide@40.0.8-alpha.5b37cb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b160899956e21fb8858411--gamut-preview.netlify.app>https://60b160899956e21fb8858411--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b160899956e21fb8858411>Deploy Logs</a></p>

**Review by saghdaey — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.6-alpha.f94fef.0<br/>@codecademy/gamut-illustrations@0.17.2-alpha.f94fef.0<br/>@codecademy/gamut-kit@0.4.6-alpha.f94fef.0<br/>@codecademy/gamut-labs@18.3.6-alpha.f94fef.0<br/>@codecademy/gamut-patterns@0.3.2-alpha.f94fef.0<br/>@codecademy/gamut-styles@10.2.2-alpha.f94fef.0<br/>@codecademy/gamut-tests@2.4.5-alpha.f94fef.0<br/>@codecademy/gamut@33.4.4-alpha.f94fef.0<br/>@codecademy/konamimojisplosion@0.2.36-alpha.f94fef.0<br/>@codecademy/markdown-overrides@0.5.48-alpha.f94fef.0<br/>@codecademy/styleguide@40.3.4-alpha.f94fef.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b7c60bf6a04110cd368bc2--gamut-preview.netlify.app>https://60b7c60bf6a04110cd368bc2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b7c60bf6a04110cd368bc2>Deploy Logs</a></p>

**Pull request merged**